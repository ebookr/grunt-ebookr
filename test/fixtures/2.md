## More testing

Even more testing