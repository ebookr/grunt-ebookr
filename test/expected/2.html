<h1 id="test">test</h1>
<p>some testing</p>
<h2 id="more-testing">More testing</h2>
<p>Even more testing</p>
